.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  docker-push:
+      strategy:
+        matrix:
+          images: [k8s-helm-deploy-image, terraform-deploy-image, python-deploy-image, tinyproxy-image]
+      runs-on: ubuntu-22.04
+      steps:
+        - uses: actions/checkout@v3
+        - name: Build
+          run: >
+            docker build -t onsdigital/${{ matrix.images }}:latest -f ${{ matrix.images }}/Dockerfile .
+        - name: Push to Docker Hub
+          run: |
+            echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+            echo "Pushing with tag [latest]"
+            docker push onsdigital/${{ matrix.images }}:latest
```

Comment:
The code formatting is off here, its indented too far. We could maybe consider adding https://editorconfig.org/ to this repo. This would help with code formatting like this. We have it in our runner repo

Reply:
Updated 👍